ansible/README-os-idr.md

```diff
@@ -10,84 +10,67 @@ Most of these scripts should also work on other platforms, providing the VM is b
 The guest must be running CentOS 7.
 
 
-Guide for the Impatient
------------------------
+Openstack: Creation of instances, volumes and security groups
+-------------------------------------------------------------
 
-Setup your OpenStack environment variables, and run:
+[Setup your OpenStack environment variables](http://docs.openstack.org/user-guide/common/cli_set_environment_variables_using_openstack_rc.html), edit the variables in `os-idr-playbooks/os-idr-create-example.yml` (especially `idr_vm_keyname` and `idr_environment`), then run:
 
-    ansible-playbook -i inventory -e omero_vm_name=FOO -e omero_vm_key_name=YOUR_KEY os-idr-uod.yml
+    ansible-playbook os-idr-playbooks/os-idr-create-example.yml
 
 
-`os-idr-playbooks/os-omero.yml`
--------------------------------
-
-This is the Ansible playbook that will be run to setup OMERO.
-You can also run it manually to install OMERO on localhost.
-
-
-`os-idr-playbooks/os-create.yml`
---------------------------------
+Openstack: Installing the IDR
+-----------------------------
 
-This playbook will connect to OpenStack and spin up a VM.
-The Ansible modules in this playbook require the `shade` Python module.
+Find the floating IP of the proxy/bastion server.
+Set `BASTION_IP` to the IP, and `IDR_ENVIRONMENT` to match the value from above.
+Run:
 
-Before running the playbook you must [setup your OpenStack environment variables](http://docs.openstack.org/user-guide/common/cli_set_environment_variables_using_openstack_rc.html).
-You can override variables at the command line, for example (note double quoting is necessary if spaces are present):
+    BASTION_IP=10.0.0.0
+    IDR_ENVIRONMENT=idr
+    ansible-playbook \
+        -i inventory/openstack-private.py \
```

Comment:
Understood, but in general, I think we may still need to re-evaluate openstack-private.py (later)